shumai/cpp/flashlight_binding.cc

```diff
@@ -534,12 +546,66 @@ unsigned* _uint64Buffer(void* t) {
   }
 }
 
-float _scalar(void* t) {
+float _float32Scalar(void* t) {
+  LOCK_GUARD
+  auto* tensor = reinterpret_cast<fl::Tensor*>(t);
+  return tensor->scalar<float>();
```

Comment:
I think we'll want `asScalar` here (which implicitly casts to the type without throwing)

https://fl.readthedocs.io/en/latest/tensor.html?highlight=scalar#_CPPv4I0ENK2fl6Tensor8asScalarE1Tv